src/index.ts

```diff
@@ -24,8 +24,19 @@ function setupOpenMRS() {
   ]);
 
   return {
-    lifecycle: getAsyncLifecycle(() => import('./root.component'), options),
-    activate: pageName,
+    pages: [
+      {
+        load: getAsyncLifecycle(() => import('./root.component'), options),
+        route: /^patient-registration/,
+      },
+      {
+        load: getAsyncLifecycle(() => import('./root.component'), {
+          featureName: 'edit-patient-details-form',
+          moduleName,
+        }),
+        route: /^patient\/.+\/edit/,
```

Comment:
I don't think this is strict enough. This would match, for example, `/patient/abc-123/vitals/edit`.

Reply:
Should I use this one then? `\/patient\/([a-zA-Z0-9\-]+)\/`

Reply:
Yes, with the `edit` at the end.

Reply:
Pushed with the new regex